optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -77,6 +77,82 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-jpa</artifactId>
```

Comment:
I suggest to only unbind the `analyze-only` execution inherited from the parent. This way, our own executions (for example in the standalone module) will stay unaffected. You can do that in the root pom using:
```xml
...
  <build>
    <pluginManagement>
        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-dependency-plugin</artifactId>
          <executions>
            <execution>
              <id>analyze-only</id>
              <phase>none</phase>
            </execution>
          </executions>
        </plugin>
...
```

Reply:
So that how you supposed to turn off plugin execution for specific executions; I was configuring the skip property. I can make it only ignore dependencies that come from certain artifacts (ex: the one from spring-boot) if you still want some analysis from the other dependencies used.